README.md

```diff
@@ -239,12 +240,16 @@ $ truckersmp-cli --wine --prefixdir "/path/to/prefix/pfx" start ets2mp
 
 ## Configuration file
 
-It's possible to set various game settings and third party programs in a configuration file. The default search path for the file is `$XDG_CONFIG_HOME/truckersmp-cli/truckersmp-cli.conf`.
+It's possible to set various game settings and third party programs in a configuration file.
 
 The configuration file format is similar to the INI format known from Windows with several sections `[section]` containing key-value-pairs `key = value`.
 All sections are optional and a key-value-pair has to be in a section.
 Boolean values can be `{yes,no}`, `{true,false}`, `{on,off}` and `{1,0}`.
 
+### Default configuration file path
+
+The default search path for the file is `$XDG_CONFIG_HOME/truckersmp-cli/truckersmp-cli.conf`.
+
```

Comment:
I actually meant to _also add_ it here: https://github.com/truckersmp-cli/truckersmp-cli#default-directories